src/app/containers/CpsRecommendations/index.jsx

```diff
@@ -18,6 +29,27 @@ const RecommendationsWrapper = styled.div`
   background-color: ${C_LUNAR};
   padding-bottom: ${GEL_SPACING};
   margin-bottom: ${GEL_SPACING_TRPL};
+  margin: ${GEL_SPACING_TRPL} 0;
+  padding: ${GEL_SPACING_DBL} ${GEL_SPACING};
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
```

Comment:
makes me think psammed grid kind of has a bug

Reply:
@ryanmccombe implemented these styling tweaks, he'd be best to respond to this

Reply:
- I've pruned the unnecessary shorthand (`padding-bottom` and `margin-bottom`) rules as these are overwritten by the lines you've highlighted. There are were no visual changes.
- We should investigate the possible Psammead bug following the merge of this PR.